.ci/lib.sh

```diff
@@ -0,0 +1,86 @@
+set -euo pipefail
```

Comment:
I found it rather clunky to get the integration tests running locally, so this includes changes I previously made to go-database-reconciler:

https://github.com/Kong/go-database-reconciler/commit/3bf528eff89a1b26eac17b67d4e399a404c16ba7